pkg/storage/storage_tests/storage_test.go

```diff
@@ -0,0 +1,71 @@
+package storagetest
+
+import (
+	"testing"
+
+	"github.com/gobuffalo/suite"
+	"github.com/gomods/athens/pkg/config/env"
+	"github.com/gomods/athens/pkg/storage"
+	"github.com/gomods/athens/pkg/storage/fs"
+	"github.com/gomods/athens/pkg/storage/mem"
+	"github.com/gomods/athens/pkg/storage/minio"
+	"github.com/gomods/athens/pkg/storage/mongo"
+	"github.com/gomods/athens/pkg/storage/rdbms"
+	"github.com/spf13/afero"
+)
+
+type StorageTests struct {
+	*suite.Model
+	storages []storage.Backend
+}
+
+func (d *StorageTests) SetupTest() {
+	ra := d.Require()
+
+	// fs
+	memFs := afero.NewOsFs()
```

Comment:
Did you intend to use `NewMemMapFs()`?

Reply:
nono this is filesystem storage. 
most of these setups are taken from its actual tests

Reply:
Oh ok, it's just the name that threw me off.

Reply:
It's funny because I named that in the test :-1: